### DISCUSSION TITLE
Duration of „Fußball Europameisterschaft 2024“

### DISCUSSION THREAD

**Discussion by FelixSFD** [Traewelling/traewelling#2566] [Questions / Support] [answered]
Hi,
Thanks for accepting my request for the EURO 2024 as an event in Träwelling. 😊
I know, the event officially starts in June, but there are already hundreds of Volunteers working for the event and travelling to the stadiums and other relevant locations to prepare the event.
Could you please extend the time range? I was not able to link my check-ins to the event today.

Best regards,

Felix

**Comment by MrKrisKrisu** [comment, marked as answer]
Sure, I've extended the timestamp so you can check in from now on.
